playground/test_playground.sh

```diff
@@ -9,8 +9,9 @@ set -u
 HOST=localhost:8080
 SELLTOKEN="0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
 BUYTOKEN="0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"
-RECEIVER="0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC"
+RECEIVER="0x94766c15b0862Dd15F9f884D85aC1AAd34199a5f"
 AMOUNT="1000000000000000000"
+PRIVATEKEY="0x93de76e801fcc65f0f517c3ca716bfc49a83a922ede9d770dd788e9e47d14f60" # cast wallet new
```

Comment:
When `anvil` starts up it prints a list of the same publicly known PKs. Would be better to use one of those here.